### PR TITLE
fix : fixing incubated from text breaking into lines

### LINKED ISSUE #4512: 🐞 Bug incubated form text breaking into lines
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

In footer section the "incubated form" text is breaking into to lines when width is below 305px

### Steps To Reproduce

1. Go to 'https://circuitverse.org/'
2. Scroll down and see the incubated-from text above the Zense logo
3. reduce the width size

### Expected Behavior

It should not break in to lines

### Screenshots

image

### Device Information [optional]

```markdown
- OS: Windows
- Browser: Chrome
- version: 120
```

### Additional context

_No response_

### Are you working on this issue?

Yes

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
Fixes #4512 

#### Describe the changes you have made in this PR -
Decreased the font-size of the text and also adjusted the padding of the div. 

### Screenshots of the changes (If any) -

Screenshot from 2024-01-07 17-33-21

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by DhairyaMajmudar** [comment]
@tanmoysrt pls .review my PR

File: app/assets/stylesheets/footer.scss [hunk 1]
```diff
@@ -72,6 +72,12 @@
 .footer-sponsor-logo-text {
   margin-bottom: 0;
   padding-bottom: 0;
+  font-size: 12.8px;
```
[the remark this answers is not in the document]
Comment (DhairyaMajmudar) [reply app/assets/stylesheets/footer.scss:75]: Sure I will update my pr with the `em` value.

[the remark this answers is not in the document]
Comment (DhairyaMajmudar) [reply app/assets/stylesheets/footer.scss]: @tanmoysrt I have updated my pr with changes suggested.

**Review by tanmoysrt — APPROVED** [review]
Looks good :+1:

**Pull request merged**